src/VisualStudio/Core/Impl/CodeModel/InternalElements/CodeParameter.cs

```diff
@@ -62,6 +62,17 @@ protected override EnvDTE.CodeElements GetCollection()
             return GetCollection<CodeParameter>(Parent);
         }
 
+        protected override string GetName()
+        {
+            return this._name;
+        }
+
+        protected override string GetFullName()
+        {
+            var node = LookupNode();
```

Comment:
Actually, it can't be null in this case. The implementation of LookupNode() in CodeParameter will throw if it can't find the ParameterSyntax from the parent member.